shared/Microsoft.AspNetCore.HttpSys.Sources/RequestProcessing/RawUrlHelper.cs

```diff
@@ -39,7 +42,8 @@ public static ArraySegment<byte> GetPath(byte[] raw)
                     if (pathStartIndex == -1)
```

Comment:
Do you see any tests verifying the comment about "http://server?query=value/1/2" being blocked by HttpSys?

Reply:
There were no tests for full uris at all. I'll try to see what happens.

Reply:
It returns a 400, which makes sense.

Reply:
Ok, please add one test.